### ISSUE TITLE
[Feature request] Act as Android's gallery app

### ISSUE THREAD

**Issue by tommytran732** [ente-io/frame#645]
The app currently does not act as Android's gallery app. Tapping on pictures in the File Manager (documentsUI) will not work if no other gallery app is installed.

I would be nice if you can have it acting as a gallery app as well.

**Comment by mnvr** [comment]
Right, this is indeed on our roadmap! https://roadmap.ente.io/set-as-default-gallery-app-p-1528/ 

Thank you for the +1. It is unfortunately not a small undertaking, but we we will try to get to this sooner indeed!
